bindings/python/test/trajectory/model/test_tabulated_trajectory.py

```
@@ -322,6 +322,84 @@ def test_get_interpolation_type(
+            },
+            output_frame=Frame.ITRF(),
+        )
+
+        assert tabulated.get_frame() == Frame.ITRF()
+
+    def test_constructor_with_null_frame_failure(
+        self,
+        test_states: list[State],
+    ):
+        # A null output frame is rejected by both explicit-frame constructors.
+        with pytest.raises(Exception):
```

Ruff (0.15.18)

[warning] Do not assert blind exception: `Exception`

(B017)